Add Turkish translation for the interface

mkykadir opened the pull request.

CarlBeek commented:
Thanks for this @mkykadir! 🙏 It is always much better to get the translation directly from someone who understands the particular technical context as opposed to a random translator.

We use a system where we stage changes via the `dev` branch before targeting master, could you please rebase off of `dev` and then we can get this merged.

mkykadir commented:
> We use a system where we stage changes via the `dev` branch before targeting master, could you please rebase off of `dev` and then we can get this merged.

Sorry, missed that; rebased and changed the PR target

CarlBeek reviewed (approved):
Thanks @mkykadir! We don't have a `CONTRIBUTING.md` or similar, so you'd have no way of knowing.

I've sanity checked this translation with Google Translate (best I can personally do in Turkish) and I'm happy.
